iglu-server/README.md

```diff
@@ -0,0 +1,204 @@
+# Iglu Server
+
+This folder contains
+
+* docker images of [Iglu Server](iglu-server)
+  * `0.3.0`
+* a docker compose example to run [Iglu Server](iglu-server) with [Postgres](https://github.com/docker-library/postgres)
+
+## Introduction
+
+[Iglu Server](iglu-server) images are based on [Snowplow base image][base-image] which leverages [the Java 8 Alpine image][alpine-image].
+
+The Iglu Server runs under [dumb-init][dumb-init] which handles reaping zombie processes
+and forwards signals on to all processes running in the container. This image also uses
+[su-exec][su-exec], as a sudo replacement, to run the Iglu Server as the non-root `snowplow` user.
+
+The container exposes the `/snowplow/config` volume to store the server configuration. If this
+folder is bind mounted then ownership will be changed to the `snowplow` user.
+
+The `-XX:+UnlockExperimentalVMOptions` and `-XX:+UseCGroupMemoryLimitForHeap` JVM options will be
+automatically provided when launching the server in order to make the JVM adhere to the memory
+limits imposed by Docker. For more information, see [this article][jvm-docker-article].
+
+Additional JVM options can be set through the `SP_JAVA_OPTS` environment variable.
+
+### Build
+
+1) Clone repository
+
+    `$ git clone git@github.com:snowplow/snowplow-docker.git`
+
+2) Navigate to `snowplow-docker/iglu-server/`
+
+    `$ cd snowplow-docker/iglu-server/`
+
+3) Build image    
+
+    `$ docker build -t iglu-server:0.x.0 0.x.0`
```

Comment:
can you use $VERSION like what has been done in 67d96e75da1b1ecf292dc189b2411d3e2e108ac6?